server/index.js

```diff
@@ -25,7 +26,6 @@ app.get('/reviews/bundle', async (req, res) => {
 
 app.post('/review', async (req, res) => {
   try {
-    console.log(req.body)
```

Comment:
Good job cleaning out the console logs in your server, I have to remind myself to do this all the time